src/main/kotlin/io/openfuture/state/controller/ExceptionHandler.kt

```diff
@@ -25,4 +24,9 @@ class ExceptionHandler {
         return ErrorDto(HttpStatus.BAD_REQUEST.value(), "Invalid parameters", fieldErrors)
     }
 
+    data class FieldErrorDto(
```

Comment:
Bud, this DTO has to be moved as well! What are the reasons to hold it here. The only ErrorDto is using it